src/pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,151 @@
-import React from "react";
+import React, { useState } from "react";
+import FindAddr from "../../components/SingUp/FindAdrr";
+import "./SignUp.scss";
 
 const SignUp = () => {
-  return <div>signup</div>;
+  const [userInfo, setUserInfo] = useState({
+    id: "",
+    pw: "",
+    pwCheck: "",
+    name: "",
+    phone: "",
+    address: "",
+    birth: "",
+  });
+
+  const [daumAddress, setDaumAddress] = useState(false);
+
+  const [valueCheck, setValueCheck] = useState(false);
+
+  const linkToDaum = () => {
+    setDaumAddress(!daumAddress);
+  };
+
+  const putUserInfo = e => {
+    setUserInfo(prev => ({ ...prev, [e.target.name]: e.target.value }));
+    console.log(userInfo);
+  };
+  // console.log(valueCheck, "valueCheck");
+
+  const alertText = () => {
+    setValueCheck(userInfo.id ? true : false);
+  };
+
+  return (
+    <form action="POST" className="signUpForm">
```

Comment:
```suggestion
    <form action="POST" className="signUp">
```
- CRA로는 index.html이 하나인 SPA프로젝트를 구성하는 것이고 이런 상황에서 결과적으로 모든 css파일들은 하나의 html에 한번에 불러와집니다.
- 결과적으로 모든 css파일이 한번에 불러와지기에 이 css파일에서 부여한 스타일이 동일한 className을 가진 다른 컴포넌트의 요소들에게도 영향을 미칠 수 있습니다.
- 따라서 반드시 **최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅** 적용해주세요

* 데이터 통신은 form태그가 아닌 fetch로 합니다.
* 불필요한 h2,div span,h3 태그까지 form으로 묶을 필요는 없습니다.
* form 태그는 사용자로부터 입력을 받을 수 있는 HTLM 입력 폼(form)을 정의할 때 사용합니다.
* <form> 요소는 다음과 같은 요소들을 하나 이상 포함할 수 있습니다.
* `<button>`,`<fieldset>`,`<input>`,`<label>`,`<option>`,`<optgroup>`,`<select>`,`<textarea>`

Reply:
네 수정했습니다 감사합니다 👍